### PR TITLE
Scrubbing up the IntelliJ Config

### PR DESCRIPTION
**Pull request by retronym**
- Add recently sprouted modules (xml and parser-combinators)
  - Replace some of the documentation with a setup script
  - Update Ant build to copy Maven sourced JARs to ./build/deps.
    These are included in the IntelliJ classpath.
  - Define the library for Ant at the project level based on
    ./lib, rather than asking the user to define global library.
  - Disable Type Aware Highlighting by default.

IntelliJ now can build everything within the IDE with CTRL-F9.

### CODE REVIEW

**Comment by retronym** [comment]
image

File: src/intellij/parser-combinators.iml.SAMPLE [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="FacetManager">
+    <facet type="scala" name="Scala">
+      <configuration>
+        <option name="compilerLibraryLevel" value="Project" />
+        <option name="compilerLibraryName" value="compiler-locker" />
+        <option name="compilerOptions" value="-sourcepath /Users/luc/scala/scala/src/reflect" />
```
Comment (adriaanm) [inline src/intellij/parser-combinators.iml.SAMPLE]: who's luc?

Comment (adriaanm) [inline src/intellij/parser-combinators.iml.SAMPLE]: Oh, I see this is covered by the README

Comment (adriaanm) [inline src/intellij/parser-combinators.iml.SAMPLE]: Is this sourcepath needed? I don't think either xml or parser-combinators requires reflect.

Comment (lrytz) [inline src/intellij/parser-combinators.iml.SAMPLE]: me! :)

but that sourcepath should only be requred for the reflect module itself, other modules (parser combinators here) should simply depend on the reflect module (if necessary)

**Comment by retronym** [comment]
Thanks for the comments. I'm still on very sporadic internet, so will address the feedback early next week.

**Comment by gkossakowski** [comment]
@retronym: ping.

**Comment by retronym** [comment]
image

Review by @adriaanm (particular build.xml changes) and @lrytz

**Comment by retronym** [comment]
Ping  @adriaanm and @lrytz

**Comment by adriaanm** [comment]
LGTM

**Pull request merged**